Source/Python/WarpXWrappers.cpp

```diff
@@ -140,6 +140,17 @@ extern "C"
         return myspc.NumRealComps();
     }
 
+    int warpx_getParticleBoundaryBufferNSpecies()
+    {
+        return WarpX::GetInstance().getBoundaryParticleBufferNSpecies();
+    }
+
+    int warpx_getNumParticlesImpactedBoundary(const char* species_name, int boundary)
+    {
+        std::string name = std::string(species_name);
```

Comment:
Stylistically, it's preferable not to repeat the type within the statement. This could read either
```c++
std::string name = species_name;
// or
auto name = std::string(species_name);
```

However, ...